src/main.js

```diff
@@ -17,3 +19,20 @@ var currentCover;
 function getRandomIndex(array) {
   return Math.floor(Math.random() * array.length);
 }
+
+//Get random image
+function getRandomImage(){
+  var randomImageIndex = getRandomIndex(covers);
+  var randomImage = covers[randomImageIndex];
+
+  return randomImage;
+}
+
+
+//Get random titles
+function getRandomTitle(){
+  var randomTitleIndex = getRandomIndex(titles);
+  var randomTitle = tiles[randomTitleIndex];
```

Comment:
There is a missing t in titles after the assign value.

Reply:
fixed